### PR TITLE
refactor: monitor-v2 package for monitor and health check runners

### PR DESCRIPTION
**Pull request by Reinis-FRP**
**Motivation**

Monitor and health check runner bots should be moved from scripts to dedicated package.

**Summary**

Moves monitor-oa, monitor-dvm and health-check-runner to new monitor-v2 package.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [x]  Untested

### CODE REVIEW

**Review by chrismaree — APPROVED** [review]
lgtm! this will clean things up a fair bit.

File: packages/monitor-v2/src/health-check-runner/index.ts [hunk 1]
```diff
@@ -39,9 +41,9 @@ if (require.main === module) {
     });
 }
 
-export function execShellCommand(cmd, env) {
+export function execShellCommand(cmd: any, env: NodeJS.ProcessEnv) {
   return new Promise((resolve) => {
-    const { stdout, stderr } = exec(cmd, { env, stdio: "inherit" }, (error, stdout, stderr) => {
+    const { stdout, stderr } = exec(cmd, { env, stdio: "inherit" }, (error: any, stdout: any, stderr: any) => {
```
Comment (mrice32) [inline packages/monitor-v2/src/health-check-runner/index.ts:46]: nit: doesn't ts/nodejs have standard types for these?

File: packages/monitor-v2/src/health-check-runner/index.ts [hunk 2]
```diff
@@ -17,7 +17,9 @@ export async function run(): Promise<void> {
 
   logger.debug({ at: "HealthCheckRunner", message: "Running health check commands", healthCheckCommands });
 
-  const outputs = await Promise.all(healthCheckCommands.map(async (command) => execShellCommand(command, process.env)));
+  const outputs = await Promise.all(
+    healthCheckCommands.map(async (command: any) => execShellCommand(command, process.env))
```
Comment (mrice32) [inline packages/monitor-v2/src/health-check-runner/index.ts:21]: Aren't the commands strings?

File: packages/scripts/src/utils/contracts.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const { getAbi, getAddress } = require("@uma/contracts-node");
+const { getAddress } = require("@uma/contracts-node");
```
Comment (mrice32) [inline packages/scripts/src/utils/contracts.ts:1]: nit: can we use import for this and the hre line below?

**Review by mrice32 — COMMENTED** [review]
Looks great! A few minor comments

File: packages/scripts/src/utils/contracts.ts (hunk 3 above)
Comment (chrismaree) [reply packages/scripts/src/utils/contracts.ts:1]: ye, that's a good point given this is ts.

**Review by chrismaree — APPROVED** [review]

**Pull request merged**